synapseAnnotations/data/tool.json

```
@@ -43,8 +43,8 @@
       },
       {
         "value": "container",
-        "description": "",
-        "source": ""
+        "description": "This is any containerized software, be it a 'workflow' or a 'tool",
```

```suggestion
        "description": "A standard unit of software that packages up code and all its dependencies so the application runs quickly and reliably from one computing environment to another.",
```

Taken from Docker site. not sure if this is appropriate for other containers , but comparing to singularity it seems to be apt (https://singularity.lbl.gov/about).